Add Usage permissions wiki article (#10303)
ppy/osu-wiki: wiki/Usage_permissions/en.md

---
tags:
  usage

>   permissions

Reviewer: tags don't really do anything if they appear as complete words in the title already

---

# Usage permissions

## Opt-in

> If you are an artist and wish to have your stance regarding whether osu! users may make use of your work known publicly or privately, please feel free to reach out to either [the osu! support team via email](mailto:support@ppy.sh) or any member of the [osu!team](/wiki/People/osu!team) with as much information as possible regarding what work you may want to allow or disallow.

Reviewer: the first clause before comma here is kind of hard to understand with how much it's trying to say... and also due to having the word "known" so far apart from the subject it's acting on ("your stance") i think. the "publicly or privately" part should be explained separately in a new sentence or something, I had to re-read this a few times to understand what was meant by that

so in general just try to simplify the sentence structure I guess. I think it's especially important for an article like this to be a clear and easy read.

Reviewer 2: "any member of the osu! team" – no
"support@ppy.sh" – no

please change this to require explicitly contacting `music@ppy.sh`, no other avenues will be accepted.

```suggestion
If you are an artist and wish to add or update your entry on this listing, please feel free to reach out to our [music licencing team](mailto:music@ppy.sh).
```

## Audio

> All tracks listed on any [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) are approved for use in the creation of osu! beatmaps. Tracks from a Featured Artist that do not appear in their listing are *not* licensed, and additional permissions may be required to use them.

Reviewer: this feels more like a *second* sentence or paragraph of this section rather than the first -- something needs to first explain what this section is generally about (the table), and then this note about FA can come in to clarify how things work in that case.

> **Important note**: external media (such as gameplay videos submitted to sites like YouTube, livestreams) is subject to different licensing requirements that must be negotiated in most circumstances with the artist themselves. Some artists have broad permissions or allowances for external media and we have made best efforts to include this information where appropriate, but please ensure you double-check using primary sources like the artist's own sites/platforms.

Reviewer 2: ```suggestion
**Important note**: Posting or streaming gameplay on other sites may potentially be subject to different licensing requirements that need to be negotiated or understood. Please do your due diligence when broadcasting gameplay to an audience.
```

Reviewer 3: Maybe one of [these notices](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#notice) that appear in the ASC should be used instead

> | Artist                                   | Status                               | Exceptions                                                  | Notes                                                                                                               |

Reviewer: maybe easier to find what you're looking for if this table were split into two, one for all the "Allowed" artists and one for everybody else

Reviewer 4: Or better split into three, one for FA, two for non-FA (allowed and disallowed). A citation of proofs (such as chat/e-mail screenshot) would be great too.

Reviewer 2: I don't understand this. Alphabetical is best.

Case: If a user is unable to type a search term, at least then they can search a single list alphabetically, rather than three.

Reviewer 4: Personally I would like to search non-FA artists randomly and listen to the song myself, since U like discovering songs in surprise. But you're right about this. If only there is a dynamic table to apply the filters in live...

Reviewer 3: I would directly not mention featured artists that have no exception, and better put a paragraph saying that featured artists that are not on this list are always allowed.

> | Artist                                   | Status                               | Exceptions                                                  | Notes                                                                                                               |

Reviewer: also tables need to be written in compact format i.e. don't try to align all of columns, just make each cell as small as possible, with 1 space of padding on each side.

| ---------------------------------------- | ------------------------------------ | ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |

> | -45                              ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/297)  |

Reviewer: `![][FA]`

> | -45                              ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/297)  |

Reviewer: if explained well enough above this table, i dont think all these duplicate notes about FA songs are necessary. it makes the mobile layout of this page terrible on top of just being kinda redundant

since that's currently where you link the fa pages, I think you could do something like this instead in the first cell
```markdown
[-45 ![][FA]](https://osu.ppy.sh/beatmaps/artists/297)
```

Reviewer 2: Agree.

| 3R2 / DJ Mashiro                 ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/187)  |
| 40mP                                     | 🔴 **Disallowed**                   |                                                             |                                                                                                                     |
| 69 de 74                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/268)  |
| 7_7                              ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/356)  |

> | a_hisa                           ![](FA) | 🟠 **Requires explicit permission** | Before uploading beatmaps, obtain the artist's permission.  | Artist can be reached via [email](mailto:hisaweb_info@yahoo.co.jp) or via [Bandcamp](https://a-hisa.bandcamp.com/). |

Reviewer: shouldnt have fa badge i think?

> | a_hisa                           ![](FA) | 🟠 **Requires explicit permission** | Before uploading beatmaps, obtain the artist's permission.  | Artist can be reached via [email](mailto:hisaweb_info@yahoo.co.jp) or via [Bandcamp](https://a-hisa.bandcamp.com/). |

Reviewer: if you take my earlier suggestion about removing all the FA notes, then the "Exceptions" and "Notes" columns can probably just be combined to compact rows like this one.

| Helblinde                        ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/5)    |
| High Tea Music                   ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/26)   |
| Hinkik                           ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/315)  |
| Hino Isuka                       ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/272)  |

> | HoneyComeBear                    ![](FA) | 🟠 Allowed, with exceptions         | Do not use or upload non-FA tracks.                         | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/319)  |

Reviewer: I would bold all of the "Status" column messages that are not simply "Allowed"

| HyuN                             ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/33)   |
| I love you Orchestra             ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/117)  |
| IAHN                             ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/3)    |
| iFeature                         ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/210)  |

> | Igorr                                    | 🔴 **Disallowed**                   |                                                             |                                                                                                                     |

Reviewer 5: small fix

```suggestion
| Igorrr                                    | 🔴 **Disallowed**                   |                                                             |                                                                                                                     |
```

| The Flashbulb                    ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/37)   |
| The Gentle Men                   ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/142)  |
| tieff                            ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/34)   |
| Tiny Waves                       ![][FA] | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/131)  |

> | tokiwa                           ![][FA] | 🟢 Allowed                          | [^monstercat-gold]                                          | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/202)  |

Reviewer 5: tokiwa is not monstercat (Tokyo Machine however is)

| YUC'e                            ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/372)  |
| yuki.                            ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/4)    |
| yukitani                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/167)  |
| Yunosuke                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/244)  |

> | Yuyoyuppe / DJ'TEKINA//SOMETHING ![](FA) | 🟠 Allowed, with exceptions         | Tracks from or themed around Touhou should not be uploaded. | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/132)  |

Reviewer: given the purpose of the page, I think the exception should be written in absolute terms rather than "should". something like "obtain the artist's permission for tracks themed around touhou" if that's what they were going for

Reviewer 2: If the artist wanted users to obtain permission, then I think that would be written. This sounds like a hard ruling (@Ephemeralis will have to confirm).

Also "should" is quite an absolute term:

![CleanShot 2023-10-30 at 03 01 22](https://github.com/ppy/osu-wiki/assets/191335/5d71fa62-b2ae-4941-bd54-3dc3e0080391)

Reviewer 4: It depends on the artist themselves, do they hard rule disallows non-FA (must) or asked first before uploading (should)? Either case is fine depending on the degree. We should confirm tk each artists whether explicit requests needed or disallowed at all.

Reviewer 2: That's not how this page works. It is opt in and already lists artists that want to be asked explicitly.

| ZxNX                             ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/288)  |

## Visual

Visual works refer to elements like beatmap backgrounds and are commonly retrieved from sites like Pixiv.

In many cases, a visual artist will make it immediately apparent if they permit the use of their work elsewhere.  All visual works should be properly attributed and provide an appropriate link-back to their source. If a direct link cannot be used for whatever reason, another link that points to the visual artist's account on their most active platform should be used instead.

If it is unclear whether a visual artist allows their work to be used on other platforms, we recommend not making use of any until appropriate usage conditions can be confirmed.

## Notes

> [^monstercat-gold]: This artist is licensed in cooperation with [Monstercat](https://osu.ppy.sh/beatmaps/artists/255), and users may be able to obtain external media usage rights via a [Monstercat Gold](https://www.monstercat.com/gold) subscription. Please consult their site for specific details on this license.

Reviewer: is this supposed to be a limitation or just a note for your convenience on where to get "external media usage rights"? it's under the "Exceptions" column of the table but it doesn't really seem like one.

Reviewer 2: The column should probably be renamed to "Notes".

> [^official-promo-exclude]: Official osu! promotional media is excluded from this limitation.

Reviewer 2: this is never used.

# Usage permissions

osu! is centred around being able to freely make and share creations with one another. It is important ensure that appropriate permissions have been acquired to use any other creator's work in new creations, and that all such work is properly attributed.

## Opt-in

For artists wishing to add or update their entry on this listing, please feel free to reach out to the osu! music licensing team at [music@ppy.sh](mailto:music@ppy.sh).

## Audio

*Notice: Posting or streaming gameplay on other sites may potentially be subject to different licensing requirements that need to be negotiated or understood. Please exercise due diligence when broadcasting gameplay to an audience.*

This section details which artists are willing to allow use of their audio works in osu!, and under what conditions (where appropriate).

All tracks listed on any [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) are approved for use in the creation of osu! beatmaps. Tracks from a Featured Artist that do not appear in their listing are *not* licensed, and additional permissions may be required to use them.

### Allowed

| Artist |  | Status |
| :-- | :-- | :-- |

> | [![][FA]](https://osu.ppy.sh/beatmaps/artists/18) | [\*namirin](https://osu.ppy.sh/beatmaps/artists/18) | ![][true] |

Reviewer 6: would make more sense to have the pill link to the [FA wiki article](https://osu.ppy.sh/wiki/en/People/Featured_Artists) instead of linking to the same artist page twice in a row.

Reviewer 7: there's less utility in almost 400 links to the fa article than links to each individual artist. between having or not having a clickable icon, this is the best option imo
